test/dynamo/test_misc.py

```diff
@@ -9872,21 +9872,24 @@ def fn(x):
 
     def test_dynamo_cache_move_to_front(self):
         class Mod(torch.nn.Module):
-            def __init__(self):
+            def __init__(self, const):
                 super(Mod, self).__init__()
                 self.fc = torch.nn.Linear(3, 3)
+                # Force Dynamo to recompile
+                self.a = const
 
             def forward(self, out):
-                return self.fc(out)
+                return self.a * self.fc(out)
 
         def fn(x, mod):
```

Comment:
Could we simplify this test and just do something like
```python
def fn(x, const):
    return x + const
```

I recall the main reason why I used a custom module here was because I based this test on `test_dynamo_cache_invalidate`, but we're not checking for invalidation here, so we don't need to use a module.